### PR TITLE
Bug 1152711 - Fix jshint error in unit test mocks

### PR DESCRIPTION
**Pull request by shamenchens**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aeb6f658a908442a8807c89f16de138ebb412dcf

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=45c4e16e022156501e62ea142b12b4d89d022072

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=062ee1bd99f9cd5a5336392115c063884bb5c065

File: apps/system/test/unit/mock_statusbar.js [hunk 1]
```diff
@@ -25,6 +28,7 @@ var MockStatusBar = {
   },
 
   updateNotification: function(count) {
+    /* jshint -W053 */
```
Comment (cctuan) [inline apps/system/test/unit/mock_statusbar.js]: What's that for?

Comment (shamenchens) [inline apps/system/test/unit/mock_statusbar.js]: Jshint is not happy with `new Number(count)`, but I think it's ok for the tests so simply ignore it.

**Pull request merged**